bin/phing.php

```diff
@@ -6,42 +6,58 @@
  * line entry point of phing located in phing.Phing
  */
 
-// Use composers autoload.php if available
-if (file_exists(__DIR__ . '/../vendor/autoload.php')) {
-    require_once __DIR__ . '/../vendor/autoload.php';
-} elseif (file_exists(__DIR__ . '/../../../autoload.php')) {
-    require_once __DIR__ . '/../../../autoload.php';
-}
+if (version_compare('7.1.0', PHP_VERSION, '>')) {
+    fwrite(
+        STDERR,
+        sprintf(
+            'This version of Phing is supported on PHP 7.1, PHP 7.2, PHP 7.3 and PHP 7.4.' . PHP_EOL .
+            'You are using PHP %s (%s).' . PHP_EOL,
+            PHP_VERSION,
+            PHP_BINARY
+        )
+    );
 
-// Set any INI options for PHP
-// ---------------------------
+    die(1);
+}
 
-/* set include paths */
-set_include_path(
-            __DIR__ . '/../classes' .
-            PATH_SEPARATOR .
-            get_include_path()
-        );
```

Comment:
Removing this lines could break the code in Phing.php line 1567
```
        if (!defined('PHP_CLASSPATH')) {
            define('PHP_CLASSPATH', get_include_path());
        }

        self::setProperty('php.classpath', PHP_CLASSPATH);
```

Reply:
The classes directory is added to the include path via composer.json